govcd/uploadtask.go

```diff
@@ -0,0 +1,33 @@
+package govcd
```

Comment:
Needs a proper Apache 2 header.  Please copy from another file.  Also any file you touch should have the copyright date 2018 added.

Reply:
Added